_includes/content/compose-extfields-sub.md

```diff
@@ -2,28 +2,33 @@ It is possible to re-use configuration fragments using extension fields. Those
 special fields can be of any format as long as they are located at the root of
 your Compose file and their name start with the `x-` character sequence.
 
-    version: '2.1'
-    x-custom:
-      items:
-        - a
-        - b
-      options:
-        max-size: '12m'
-      name: "custom"
+```none
```

Comment:
not a big deal, but this could be \`\`\`yaml instead

Reply:
Hey thanks @shin- I'll fix that.

Reply:
There are some instances when the yaml codefence breaks Atom syntax highlighting, but mostly when the YAML has pseudo-stuff in it (like `...` to show that output is truncated). It never breaks rendering of the actual output but can be annoying when editing the file. In those cases you will notice that syntax highlighting gets weird and you can selectively change those few back to `none`.